is-palindrome.js

```diff
@@ -0,0 +1,15 @@
+// Napisz funkcję, isPalindrome(), która sprawdzi, czy podane słowo jest palindromem.
+
+function isPalindrome(string) {
+    const palind = string.split('').reverse().join('');
```

Comment:
`palind` is a bad name for this constant. It's a reversed string, name it `reversedString`. It's not always a palindrome.